CONTRIBUTING.md

```diff
@@ -1,5 +1,29 @@
-# Bug Reports & Feature Requests
+# List of rules and norms
+
+## Bug Reports & Feature Requests
 Bug reports and feature requests are greatly appreciated and can be done in form of an [issue](https://github.com/JakeRoggenbuck/auto-clock-speed/issues)
 
-# Develop
+## Develop
 If you would like to help complete a task, comment in the issue and we may assign it to you!
+
+## Pull Requests
+- Pull requests must be tested but with `cargo test` and ran on local machine (laptop with battery)
+- Pull requests that change core functionality must have a review
+
+## Issues marked with `help wanted` or `good first issue`
+- Issues marked with either `help wanted` or `good first issue` are a great place to start!
+- Issues marked with these are a great place for discussion and user suggestions
+
+## Projects tab
+- The [projects tab](https://github.com/JakeRoggenbuck/auto-clock-speed/projects/1) is something we use extensively to organize when things get done
+
+## Milestones & Version
+- Milestones are how we prepare for a release and version number increment
+- We will add 10-20 issues to a milestone to be worked on
+- Issue move around milestones as we change our requirements and priorities
+- Bugs always get added to the closet milestone
+
+## Releases
+- Before a release, we test on several computers making sure they all function, and all function the same
+- We add a git tag to the branch and push it
+- Then we draft and publish the release
```

Comment:
Maybe say roughly how long will the program be updated and new version released (ie every few months, every few weeks, every minute)